Lib/sndhdr.py

```diff
@@ -72,6 +80,7 @@ def whathdr(filename):
 
 tests = []
 
+
```

Comment:
Unrelated change.

Reply:
This was a pep8 recommendation from a linting tool that I use. I'll remove it.